docs/source/programming_resources/tutorial_specific/onbot_java/controlling_a_servo/Controlling-a-Servo-(OnBot-Java).rst

```diff
@@ -1,5 +1,5 @@
-Controlling a Servo (OnBot Java)
-================================
+Controlling a Servo :bdg-info:`OBJ`
+====================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".